sdk/communication/azure-communication-sms/src/samples/java/com/azure/communication/sms/samples/quickstart/ReadmeSamples.java

```diff
@@ -39,56 +40,82 @@ public static void main(String[] args) {
         // Build a new SmsClient
         SmsClient smsClient = smsClientBuilder.buildClient();
 
-        // Currently Sms services only supports one phone number
-        List<PhoneNumberIdentifier> to = new ArrayList<PhoneNumberIdentifier>();
-        to.add(new PhoneNumberIdentifier("<to-phone-number>"));
-
-        // SendSmsOptions is an optional field. It can be used
         // to enable a delivery report to the Azure Event Grid
         SendSmsOptions options = new SendSmsOptions();
         options.setEnableDeliveryReport(true);
 
-        // Send the message and check the response for a message id
-        SendSmsResponse response = smsClient.sendMessage(
-            new PhoneNumberIdentifier("<leased-phone-number>"), 
-            to, 
+    }
+
+    public void sendMessageToOneRecipient (SmsClient smsClient, SendSmsOptions options ){
```

Comment:
Lets create the options here to show the usage.. set both enableDeliveryReport and tag